packages/@lwc/babel-plugin-component/src/dynamic-imports.ts

```diff
@@ -72,6 +72,7 @@ export default function (): Visitor<LwcBabelPluginPass> {
             if (loader) {
                 const loaderId = getLoaderRef(path, loader, state);
                 path.replaceWith(loaderId);
+                incrementMetricCounter(CompilerMetrics.DynamicImportTransform, state);
```

Comment:
Example metric for getting how often we transform dynamic imports.

Is this useful data to have? Or is there a better feature to instrument in the babel-plugin?

Reply:
It is in the wishlist (https://github.com/salesforce/lwc/issues/2964), so I assume this is something we'd want to get rid of someday? Hence it's useful to know how often we're hitting it? I'm not super familiar with the specifics of this though.

Reply:
Yeah that's a good candidate. We'll have to update the [existing](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-core/src/framework/decorators/register.ts#L140) [guards](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-core/src/framework/decorators/register.ts#L216) so it can run in prod mode so I'll save it for a subsequent PR.